pages/common/conda-create.md

```diff
@@ -0,0 +1,16 @@
+# conda create
+
+> Create new conda environments.
+> More information: <https://docs.conda.io/projects/conda/en/latest/commands/create.html>
+
+- Create a new environment named py35 and install Python 3.5 and numpy at least 1.11 in it:
+
+`conda create --yes --name py35 python=3.5 "numpy>=1.11"`
+
+- Make exact copy of an environment:
+
+`conda create --clone py35 --name py35-2`
```

Comment:
Only adding `-2` might be a tish hard to notice with the preceeding `35`, perhaps `py35-copy` would be a better name?
```suggestion
`conda create --clone {{py35}} --name {{py35-copy}}`
```